infra_monitored_account/app.py

```diff
@@ -1,10 +1,57 @@
 #!/usr/bin/env python3
 import os
+import sys
 
 import aws_cdk as cdk
 
+sys.path.append("../src")
 from infra_monitored_account.infra_monitored_stack import InfraMonitoredStack
 
+from lib.core import file_manager
+from lib.core.constants import SettingFileNames
+from lib.settings.settings_reader import (
+    GeneralSettingsReader,
+    MonitoringSettingsReader,
+    RecipientsSettingsReader,
+)
+from lib.settings import settings_validator
+
+
+def read_settings():
```

Comment:
Can leave it here or create another utility module in src/settings to have something that can accept files path, read and validate all files from there. Do you think it's a good idea?